### DISCUSSION TITLE
Can't setup 2-factor identification and I'm about to lose my account

### DISCUSSION THREAD

**Discussion by Yntec** [community/community#132599] [New to GitHub]
I can't scan the QR code or receive a SMS because I don't have a smartphone and the 'Your two-factor secret' code does nothing when I paste it into the Verify the code from the app box because it is expecting 6 characters and the setup key has 16 characters.

**Comment by Chamal1120** [comment]
First, SMS feature works on normal keypad dumb phones which are pretty cheap. If your account worth that amount buy a dumb phone.

If not, install an Android emulator and install a open source TFA app to that.

Hope this helps ya

**Comment by ghostinhershell** [comment]
Hi @Yntec, thanks so much for reaching out in our community. 

We do have a support article that documents next steps if you lose access to your two-factor authentication credentials: [Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). I think this section, specifically, should be helpful: [Requesting help with two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#requesting-help-with-two-factor-authentication).

If that doesn't help, you can submit a support ticket via our [Support page](https://support.github.com/contact).
